README.md

```diff
@@ -0,0 +1,94 @@
+# Elasticsearch Plugin
+
+This is a plugin for [Kubemove](https://github.com/kubemove/kubemove) to sync data between two Elasticsearch clusters deployed with [ECK Operator](https://github.com/elastic/cloud-on-k8s).
+
+## Pre-requisite
+
+You must install [Kubemove](https://github.com/kubemove/kubemove) installed before installing this plugin.  In order to install Kubemove framework, please for the guide [here](https://github.com/kubemove/kubemove#deploy-kubemove).
+
+## Install Plugin
```

Comment:
Let's add a supported repository column for elasticsearch since we are supporting s3 base repo as of now.

Reply:
done